ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -385,6 +392,7 @@ public void shutdown() throws StoreException {
       checkStarted();
       try {
         logger.info("Store : " + dataDir + " shutting down");
+        compactor.close(2);
```

Comment:
This is to help tests which otherwise have to wait for a fixed timeout before proceeding which increases test run time. In the case of log or index there aren't any other threads that they have to wait for before closing so I don't think what applies to them should necessarily apply here.